.github/dependabot.yml

```diff
@@ -47,6 +49,8 @@ updates:
       # Kafka
       - dependency-name: org.apache.kafka:*
       - dependency-name: org.apache.zookeeper:zookeeper
+      - dependency-name: org.xerial.snappy:snappy-java
+      - dependency-name: io.strimzi:kafka-oauth-client
```

Comment:
Same here?

Reply:
We want this one. We have tests verifying that it's working. If the PR build fails, we need to ping the Stirmizi team and investigate (we had issues in the past).